include/items.php

```diff
@@ -185,8 +185,7 @@ function add_page_info_to_body($body, $texturl = false, $no_photos = false) {
 			$body = $removedlink;
 		}
 
-		$url = str_replace(['/', '.'], ['\/', '\.'], $matches[1]);
-		$removedlink = preg_replace("/\[url\=" . $url . "\](.*?)\[\/url\]/ism", '', $body);
+		$removedlink = preg_replace("/\[url\=" . preg_quote($matches[1]) . "\](.*?)\[\/url\]/ism", '', $body);
```

Comment:
Is this intended to be in this PR or is it a github issue that this change in item.php was included here?

Reply:
Nope, this has been fixed with a rebase.